### ISSUE TITLE
Support GGUF models

### ISSUE THREAD

**Issue by vladfaust** [runpod-workers/worker-vllm#98]
See https://github.com/vllm-project/vllm/issues/1002, https://github.com/vllm-project/vllm/pull/5191. 

Should be able to set `gguf` as `QUANTIZATION` envar, but we also need to provide exact quant. I'm thinking of some `MODEL_FILENAME` envar containing the exact filename in the model's repository. The model download logic shall be changed, see https://github.com/Isotr0py/vllm/blob/main/examples/gguf_inference.py.

**Comment by nerdylive123** [comment]
yeah i guess now we dont need set quantization env var but we need to support it like from this one
![image](https://github.com/user-attachments/assets/56e5f88f-2446-4e59-b76e-a182663398ff)

**Comment by mohamednaji7** [comment]
@nerdylive123 How could I implement GGUF for my fork?

**Comment by Paladinium** [comment]
@pandyamarut : What was the reason to close this issue?
